samples/excel/85-preview-apis/culture-info-date-time.yaml

```diff
@@ -0,0 +1,138 @@
+            console.log(`  System short date format: ${systemShortDatePattern}`);
+            console.log(`  System date separator: ${systemDateSeparator}`);
+            console.log(`  System long time format: ${systemLongTimePattern}`);
+            console.log(`  System time separator: ${systemTimeSeparator}`);
+
+            await context.sync();
+          });
+        }
+
+        /* Use the cultural settings API to write date and time settings in your table. */
+        async function writeDateTimeSetting() {
+          await Excel.run(async (context) => {
+            context.application.cultureInfo.datetimeFormat.load([
+              "longDatePattern",
+              "shortDatePattern",
+              "dateSeparator",
+              "longTimePattern",
+              "timeSeparator"
+            ]);
+            await context.sync();
+
+            // System settings are read-only and set in the user's operating system.
+            const systemLongDatePattern = context.application.cultureInfo.datetimeFormat.longDatePattern;
+            const systemShortDatePattern = context.application.cultureInfo.datetimeFormat.shortDatePattern;
+            const systemDateSeparator = context.application.cultureInfo.datetimeFormat.dateSeparator;
+            const systemLongTimePattern = context.application.cultureInfo.datetimeFormat.longTimePattern;
+            const systemTimeSeparator = context.application.cultureInfo.datetimeFormat.timeSeparator;
+
+            const sheet = context.workbook.worksheets.getActiveWorksheet();
+            const longDate = sheet.getRange("A2:B2");
+            longDate.values = [["Long date", systemLongDatePattern]];
+            const shortDate = sheet.getRange("A3:B3");
+            shortDate.values = [["Short date", systemShortDatePattern]];
+            const shortDate = sheet.getRange("A4:B4");
+            shortDate.values = [["Date separator", systemDateSeparator]];
+            const shortDate = sheet.getRange("A5:B5");
+            shortDate.values = [["Long time format", systemLongTimePattern]];
+            const shortDate = sheet.getRange("A6:B6");
+            shortDate.values = [["Time separator", systemTimeSeparator]];
+
+            sheet.tables
+              .getItemAt(0)
+              .getRange()
+              .format.autofitColumns();
+
+            await context.sync();
+          });
+        }
+
+        /** Default helper for invoking an action and handling errors. */
+        async function tryCatch(callback) {
+          try {
+            await callback();
+          } catch (error) {
+            // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+            console.error(error);
+          }
+        }
+    language: typescript
+template:
+    content: |-
+        -| <section class="ms-font-m"> <p>This sample shows how to apply the cultural settings APIs to retrieve read-only operating system date and time settings.</p> </section>
```

Comment:
I think the "-|" at the beginning of this line is unintentional and can be removed.